AGPs/AGP-F.md

```diff
@@ -0,0 +1,49 @@
+---
+AGP: N/A
+Title: Panvala Launch Partnership
+Author: Niran Babalola (niran)
+Status: Stage III
+Track: Finance
+Created: 2019-07-09
+---
+
+# Panvala Launch Partnership
+
+## Address of the transfer recipient
+The address of Panvala's token capacitor smart contract will be determined when
+it is deployed on mainnet. Instead of a direct transfer, this proposal gives
+the Aragon Association the authority to make transfers over time.
+
+## Amount of the transfer
+Up to $2,000,000.
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+Up to 24 monthly transfers of $83,333.33 each. Contributions can be made for up to three months
+at a time, but without allowing a month to pass after any of the individual
+scheduled contributions. These recurring transfers can be stopped at the
+discretion of either the Aragon Association or Aragon token holders.
+
+## Purpose of the transfer
+
+Panvala is a donor-driven platform that helps fund the work that the whole Ethereum community depends on. You can think of it as a decentralized version of the Ethereum Foundation that runs on its own token. Details of Panvala’s goals and design are available in the [Panvala Whitepaper](https://v.fastcdn.co/u/b75396d2/44139515-0-Panvala-Whitepaper-J.pdf).
+
+In Panvala’s [first batch of grants](https://medium.com/@Panvala/twelve-grants-awarded-in-batch-one-of-panvala-token-grants-59b8df7422fe) released on February 1, Aragon One earned 300,000 PAN for their work on the [Voting Relay Protocol](https://forum.aragon.org/t/simple-voting-relay-protocol-optimistic-vote-tallying/473), a layer 2 protocol to scale token voting on Ethereum. Those tokens will be included in the initial token distribution when Panvala launches on the mainnet in August.
+
+10,000,000 PAN have been set aside to divide between Panvala Launch Partners, selected grant recipients who have committed to donate pan they earn or purchase from other grantees over the first two years of Panvala. Launch Partners receive voting power for fulfilling their pledge. This AGP proposes giving the Aragon Association permission to donate up to 2 million USD over two years to Panvala on behalf of Aragon One. The final amount will be chosen by the Aragon Association as they see fit.
```

Comment:
> This AGP proposes giving the Aragon Association permission to donate up to 2 million USD over two years to Panvala on behalf of Aragon One.

Is the AA actually donating actual USD or are they donating PAN that Aragon One and other grant recipients receive and then give back to the AA (presumably as some condition for receiving funding from the AA)?

Reply:
Panvala's smart contracts can only handle PAN. The AA donates PAN earned by grant recipients to Panvala. During periods when the AA does not have enough PAN to fulfill the pledge, they use the treasury to acquire PAN to fulfill the pledge, or walk away from the pledge and the remaining tokens at the discretion of the AA or ANT holders.